.github/workflows/unit-tests.yml

```diff
@@ -9,4 +9,14 @@ jobs:
     - name: Install AssemblyScript Dependencies
       run: cd runtime && yarn install
     - name: Run tests
-      run: cd runtime && yarn run test:ci
\ No newline at end of file
+      run: cd runtime && yarn run test:ci
+  unit-tests-spec-builder:
```

Comment:
maybe we can change the name of the jobs from `unit-tests` and `unit-tests-spec-builder` to:
`runtime` and `spec-builder`